### ISSUE TITLE
Request for Grade

### ISSUE THREAD

**Issue by baut0015** [baut0015/saas-website#2]
<img width="912" alt="Screen Shot 2020-04-21 at 1 44 40 AM" src="https://user-images.githubusercontent.com/55094055/79930002-5eed2180-8415-11ea-9821-398337811a0a.png">
<img width="912" alt="Screen Shot 2020-04-21 at 1 45 04 AM" src="https://user-images.githubusercontent.com/55094055/79930006-60b6e500-8415-11ea-9f9f-9eedef14688c.png">
<img width="912" alt="Screen Shot 2020-04-21 at 1 45 26 AM" src="https://user-images.githubusercontent.com/55094055/79930007-614f7b80-8415-11ea-8d9f-f3455bededad.png">
@muxlowc

**Comment by muxlowc** [comment]
@baut0015 You have to also write a bit of a rationale, and request a letter grade :)
https://learn-the-web.algonquindesign.ca/courses/web-dev-2/week-15/#request-for-grade

**Comment by baut0015** [comment]
**Rationale**

I believe this site reflects my original sketches. I'm pleased with the result and I have shown good examples of what we have learned throughout the whole semester, with gridifier, modulifier settings adding the pin and buttons and the type.  

The grid in the _how to use_ section was the hardest since I had to reorder and create grids inside a grid with the button looking just like my sketches but in the end, I'm proud of my problem solving skills. My website works as intended in the devices like my phone, however when I write in the forms through my phone the keyboard pops in the middle of my phone and it kind of bugs out and it is for this very reason that I would give myself a A-. 

The nav had to be re-designed since the typography setting that I chose wouldn't allow me to have the nav beside the logo as the Sign Up button would cut in half, so in the end I just popped the nav below the logo section, making it pleasing to look at. 

Overall my whole concept looks just as how I envision it. I'm proud of the responsiveness through the many screens and my design works harmoniously with the website. The gridifier, modulifier and type presets make it for an easier time in coding. Thank you for this year of learning.

@muxlowc
